<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileBrowser</class>
 <widget class="QMainWindow" name="FileBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Linux File Explorer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Downloads/Explorer.ico</normaloff>../../../Downloads/Explorer.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="commandEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="param1Edit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="param2Edit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QListView" name="listView">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="cursor" stdset="0">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Address bar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBack"/>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
   <addaction name="actionHome"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Command bar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionCd"/>
   <addaction name="actionMkdir"/>
   <addaction name="actionRmdir"/>
   <addaction name="separator"/>
   <addaction name="actionCp"/>
   <addaction name="actionMv"/>
   <addaction name="actionRm"/>
   <addaction name="separator"/>
   <addaction name="actionPwd"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHome">
    <property name="title">
     <string>Home</string>
    </property>
    <addaction name="actionRefresh"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuCommands">
    <property name="title">
     <string>Commands</string>
    </property>
    <addaction name="actionCd"/>
    <addaction name="actionMkdir"/>
    <addaction name="actionRmdir"/>
    <addaction name="separator"/>
    <addaction name="actionCp"/>
    <addaction name="actionMv"/>
    <addaction name="actionRm"/>
    <addaction name="separator"/>
    <addaction name="actionPwd"/>
   </widget>
   <addaction name="menuHome"/>
   <addaction name="menuCommands"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_folder">
   <property name="icon">
    <iconset>
     <normaloff>../../../Downloads/Folder-Add (1).ico</normaloff>
     <normalon>../../../Downloads/Folder-Add-32.png</normalon>../../../Downloads/Folder-Add (1).ico</iconset>
   </property>
   <property name="text">
    <string>New folder</string>
   </property>
  </action>
  <action name="actionMove_to">
   <property name="text">
    <string>Move to...</string>
   </property>
  </action>
  <action name="actionCopy_to">
   <property name="text">
    <string>Copy to...</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset>
     <normaloff>../../../Downloads/Close.ico</normaloff>../../../Downloads/Close.ico</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset>
     <normaloff>../../../Downloads/Google Chrome/gfx/9002.png</normaloff>../../../Downloads/Google Chrome/gfx/9002.png</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset>
     <normaloff>../../../Downloads/Google Chrome/gfx/9023.png</normaloff>../../../Downloads/Google Chrome/gfx/9023.png</iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
  </action>
  <action name="actionCd">
   <property name="text">
    <string>cd</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;change directory&lt;/span&gt;&lt;br/&gt;cd &lt;span style=&quot; font-style:italic;&quot;&gt;dirname&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Alt+1</string>
   </property>
  </action>
  <action name="actionMkdir">
   <property name="text">
    <string>mkdir</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;make directory&lt;/span&gt;&lt;br&gt;mkdir &lt;span style=&quot; font-style:italic;&quot;&gt;dirname&lt;/span&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Alt+2</string>
   </property>
  </action>
  <action name="actionRmdir">
   <property name="text">
    <string>rmdir</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;remove directory&lt;/span&gt;&lt;br&gt;rmdir &lt;span style=&quot; font-style:italic;&quot;&gt;dirname&lt;/span&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Alt+3</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset>
     <normaloff>../../../Downloads/Google Chrome/gfx/9044.png</normaloff>../../../Downloads/Google Chrome/gfx/9044.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset>
     <normaloff>../../../Downloads/Google Chrome/gfx/9359.png</normaloff>../../../Downloads/Google Chrome/gfx/9359.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="toolTip">
    <string>Home</string>
   </property>
  </action>
  <action name="actionCp">
   <property name="text">
    <string>cp</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;copy file&lt;/span&gt;&lt;br&gt;cp &lt;span style=&quot; font-style:italic;&quot;&gt;filename1 filename2&lt;/span&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Alt+4</string>
   </property>
  </action>
  <action name="actionMv">
   <property name="text">
    <string>mv</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;move file&lt;/span&gt;&lt;br&gt;mv &lt;span style=&quot; font-style:italic;&quot;&gt;filename1 filename2&lt;/span&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Alt+5</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionClose_current_tabs">
   <property name="text">
    <string>Save tabs</string>
   </property>
  </action>
  <action name="actionRm">
   <property name="text">
    <string>rm</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;remove file&lt;/span&gt;&lt;br&gt;rm &lt;span style=&quot; font-style:italic;&quot;&gt;filename&lt;/span&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Alt+6</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionPwd">
   <property name="text">
    <string>pwd</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;print working directory&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Alt+7</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
